test/pop_manager_test.erl

```diff
@@ -193,5 +243,52 @@ fancy_test() ->
 
     ?assert(no_more_messages()),
 
+    %% Test request_block_hash_range
+
+    ReqHashRangeTestFn = 
+	fun(RequestList, State, ResultList) ->
+		StateM = pop_manager:on_net_message(self(), Time, request_block_hash_range, get_hashes_tuple(RequestList), State),
+
+		?assertEqual(StateM, State),
```

Comment:
Is this is to ensure request_block_hash_range doesn't change pop_manager state?

Reply:
ya, pretty clear from the code that it doesn't but just a quick sanity check